runtime/testy/stateManager.py

```diff
@@ -44,17 +55,15 @@ def start(self):
     # And that there are the correct number of elements
     while True:
       request = self.input.get(block=True)
-      cmd_type = request[0]
-      if cmd_type == SM_COMMANDS.RESET:
-        self.initRobotState()
-      elif cmd_type == SM_COMMANDS.ADD:
-        self.addPipe(request[1], request[2])
-      elif cmd_type == SM_COMMANDS.GET_VAL:
-        self.getValue(request[1])
-      elif cmd_type == SM_COMMANDS.SET_VAL:
-        self.setValue(request[1], request[2])
-      elif cmd_type == SM_COMMANDS.HELLO:
-        print("HELLO")
-      # TODO: Add better error description
-      else:
+
+      if(len(request) != 2):
+        self.badThingsQueue.put(BadThing(sys.exc_info(), "Wrong input size, need list of size 2" % (request,), event = BAD_EVENTS.UNKNOWN_PROCESS, printStackTrace = False))
+      elif(not isinstance(request[0], SM_COMMANDS)):
```

Comment:
request[0] should be cmdType

question. why don't you just use `if cmdType not in self.commandMapping`?